resources/otoole.yaml

```diff
@@ -142,7 +142,7 @@ OperationalLifeStorage:
     indices: [REGION,STORAGE]
     type: param
     dtype: float
-    default: 10
+    default: 1
```

Comment:
Was this change deliberate? Doesn't matter either way as OL is defined within the workflow was just wondering

Reply:
Was deliberate, and I just aligned it with powerplant operational life default values